Dockerfiles/midstream/run_tests.py

```diff
@@ -87,13 +100,15 @@ def test_for_extract_and_validate_bundle_image(self):
         return 0
 
 if __name__ == '__main__':
-    resultcodes = [0, 50, 70, 102, 103]
+    import time
+    time.sleep(3600)
+    resultcodes = [0, 50, 70, 102]
     # run fix_etc_passwd.sh to setup random user generated in openshift
-    subprocess.call(['sh', "/usr/bin/fix_etc_passwd.sh"])
+    subprocess.call([PLAYBOOKS_DIR+"/Dockerfiles/midstream/fix_etc_passwd.sh"])
     runTest = RunMidstreamCVPTests()
-    with open("tests_result.log", 'w') as f:
+    with open("/tmp/tests_result.log", 'w') as f:
```

Comment:
Why `/tmp` is hardcoded ?

Reply:
since the test log is being generated and renamed in original logic. 
However, its no longer relevant.